### ISSUE TITLE
[Flaky Test] Test ML TimeoutError: Timeout waiting for 'cond' to be true. Waited 60 seconds.

### ISSUE THREAD

**Issue by kaiyan-sheng** [elastic/beats#10731]
## Flaky Test
I've seen this twice:
1. https://travis-ci.org/elastic/beats/jobs/492470715#L1419
2. https://travis-ci.org/elastic/beats/jobs/492654041#L1412

* **Test Name:** 
test_ml.Test.test_ml_setup_0
https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-linux/5033/beat=filebeat,label=linux-immutable/testReport/junit/test_ml/Test/test_ml_setup_0/
test_ml.Test.test_ml_setup_1
https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-linux/5033/beat=filebeat,label=linux-immutable/testReport/junit/test_ml/Test/test_ml_setup_0/

* **Branch:** 7.0

### Stack Trace

```
======================================================================
ERROR: Test ML are installed in all possible ways [with modules_flag=False]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/go/src/github.com/elastic/beats/filebeat/build/python-env/local/lib/python2.7/site-packages/parameterized/parameterized.py", line 392, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/go/src/github.com/elastic/beats/filebeat/tests/system/test_ml.py", line 45, in test_ml_setup
    self._run_ml_test(modules_flag)
  File "/go/src/github.com/elastic/beats/filebeat/tests/system/test_ml.py", line 125, in _run_ml_test
    max_timeout=60)
  File "/go/src/github.com/elastic/beats/filebeat/tests/system/../../../libbeat/tests/system/beat/beat.py", line 348, in wait_until
    "Waited {} seconds.".format(max_timeout))
TimeoutError: Timeout waiting for 'cond' to be true. Waited 60 seconds.
-------------------- >> begin captured stdout << ---------------------
Using elasticsearch: http://elasticsearch:9200
Test modules_flag: False
render config
--------------------- >> end captured stdout << ----------------------
======================================================================
ERROR: Test ML are installed in all possible ways [with modules_flag=True]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/go/src/github.com/elastic/beats/filebeat/build/python-env/local/lib/python2.7/site-packages/parameterized/parameterized.py", line 392, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/go/src/github.com/elastic/beats/filebeat/tests/system/test_ml.py", line 45, in test_ml_setup
    self._run_ml_test(modules_flag)
  File "/go/src/github.com/elastic/beats/filebeat/tests/system/test_ml.py", line 125, in _run_ml_test
    max_timeout=60)
  File "/go/src/github.com/elastic/beats/filebeat/tests/system/../../../libbeat/tests/system/beat/beat.py", line 348, in wait_until
    "Waited {} seconds.".format(max_timeout))
TimeoutError: Timeout waiting for 'cond' to be true. Waited 60 seconds.
-------------------- >> begin captured stdout << ---------------------
Using elasticsearch: http://elasticsearch:9200
License already enabled
Test modules_flag: True
render config
--------------------- >> end captured stdout << ----------------------```

**Comment by kaiyan-sheng** [comment]
I see https://github.com/elastic/beats/pull/9523 increased timeout from 30s to 60s but it's still not enough?

**Comment by kaiyan-sheng** [comment]
I think this might be related to https://github.com/elastic/beats/pull/10732, not timeout.

**Comment by ruflin** [comment]
I think that https://github.com/elastic/beats/pull/10723 solved this? Should we close this one?

**Comment by kaiyan-sheng** [comment]
The fix is backported into 7.0 branch. Thanks!
